web/modules/custom/dpl_admin/dpl_admin.module

```diff
@@ -216,7 +216,54 @@ function _dpl_admin_form_alter_node_page(array &$form, FormStateInterface $form_
     $form['field_hero_title']['#states']['invisible'] = $inactive_state;
     $form['field_subtitle']['#states']['invisible'] = $inactive_state;
   }
+}
 
+/**
+ * Implements hook_form_FORM_ID_alter() for the user add/edit form.
+ *
+ * Re-ordering the user edit form, to make the password demands clearer.
+ *
+ * @param array<mixed> $form
+ *   See the $form in dpl_admin_form_alter().
+ */
+function _dpl_admin_form_alter_user_form(array &$form, FormStateInterface $form_state, string $form_id): void {
+  $form['account']['current_pass']['#weight'] = 997;
+  $form['account']['pass']['#weight'] = 998;
+  $form['account']['password_policy_status']['#weight'] = 999;
+}
+
+/**
+ * Implements hook_module_implements_alter().
+ *
+ * Make sure the dpl_admin form_alter happens last, after any other contrib
+ * modules. This is necessary for us to alter the password policy position.
+ */
+function dpl_admin_module_implements_alter(array &$implementations, string $hook): void {
+  if ($hook == 'form_alter') {
+    $group = $implementations['dpl_admin'];
+    unset($implementations['dpl_admin']);
+    $implementations['dpl_admin'] = $group;
+  }
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Add basic styling classes to the "reset password" page, as it does not
+ * have any Gin styling.
+ */
+function dpl_admin_form_user_pass_reset_alter(array &$form, FormStateInterface $form_state, string $form_id): void {
+  $form['#attributes']['class'][] = 'rich-text';
+  $form['#attributes']['class'][] = 'dpl-form';
+  $form['title'] = [
+    '#type' => 'markup',
+    '#prefix' => '<h1>',
+    '#suffix' => '</h1>',
+    '#markup' => t('Reset password', [], ['context' => 'DPL admin']),
+    '#weight' => '-999',
+  ];
+
+  $form['actions']['submit']['#attributes']['class'] = ['btn-primary', 'btn-filled btn-small', 'dpl-button'];
```

Comment:
Please consider if there is some configuration we have forgotten to add.

We use the [Gin Login module](https://www.drupal.org/project/gin_login) which already "provides a nice login screen based on either Claro or Gin for User Login, User Register, User Forgot Password". 

I think it would be appropriate if this also supports the "Reset password page". If not I think it would be a worthy contribution.

Reply:
I get the confusion. `gin_login` does provide styling for /user/password, but what I'm styling is the page you end up on, when you're clicking on the link that Drupal sends to your email:

<img width="1077" alt="Screenshot 2024-12-03 at 12 01 05" src="https://github.com/user-attachments/assets/c8edd065-cf92-41f1-971b-5b0a5f4d3a26">

`/user/reset/1/XXXXXXX`

Reply:
Thanks for clarifying.

I still think it would be a worthy contribution to add support for the "Reset password page" for Gin Login instead of keeping this in the frontend theme.

Reply:
I've added an issue here:
https://www.drupal.org/project/gin_login/issues/3491373

Are you saying that I should also create a patch, as part of this pull request/task?

Reply:
That was what I had hoped for compared to the current solution, but I do not know how difficult it would be to achieve :smile: